kernels/simple_copy/main.c

```diff
@@ -47,5 +47,7 @@ int main() {
       nerr += 1;
   }
 
-  return nerr;
+  if (nerr > 0)
+    return 1;
+  return 0;
```

Comment:
If you want, you can do this with a oneliner
```suggestion
  return nerr != 0;
```